lib/rubocop/cop/lint/interpolated_string.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for interpolation in a single quoted string.
+      #
+      # @example
+      #
+      #   # bad
+      #
+      #  foo = 'something with #{interpolation} inside'
+      #
+      # @example
+      #
+      #   # good
+      #
+      #  foo = "something with #{interpolation} inside"
+      class InterpolationCheck < Cop
+        MSG = 'Did you want to interpolate?'.freeze
+
+        def on_str(node)
+          offense_source_range = regex_match_group(node, /(#\{.*\})/)
+          return unless offense_source_range
+          add_offense(node, offense_source_range, MSG)
```

Comment:
`MSG` is redundant here. (RuboCop will tell you this if you're running latest `master` and run it.)